src/index.ts

```diff
@@ -177,6 +237,8 @@ class Logging {
     this.auth = new GoogleAuth(options_);
     this.options = options_;
     this.projectId = this.options.projectId || '{{projectId}}';
+    this.configService = new v2.ConfigServiceV2Client(this.options);
```

Comment:
This feels like a strange thing to include as part of this review.  Can you explain why we need these to support promise first?

Reply:
I will present this in the follow up of smaller PRs